src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -373,7 +373,7 @@ public class CoinstakeContext
             this.targetReserveBalance = 0; // TOOD:settings.targetReserveBalance
             this.stakeProgressFlag = StakeNotInProgress;
 
-            this.posConsensusValidator = consensusLoop.Validator as IPosConsensusValidator;
+            //this.posConsensusValidator = consensusLoop.Validator as IPosConsensusValidator;
```

Comment:
Remove commented code please if it's not needed

Reply:
Good catch - what shall I do with the following lines in NLog config....

I just did a global search and found it.....

<logger name="Stratis.Bitcoin.Features.Consensus.PowConsensusValidator" minlevel="Trace" writeTo="debugConsensusValidationFile" />

Reply:
since there is this line below it - I think we can delete the above lines for pos and pow?

`<logger name="Stratis.Bitcoin.Features.Consensus.Rules.*" minlevel="Trace" writeTo="debugConsensusValidationFile" />`